aspnetcore/getting-started.md

```diff
@@ -23,6 +23,8 @@ uid: getting-started
     cd aspnetcoreapp
     dotnet new web
     ```
+    
+    Note: If you get an error running `dotnet new web` or you must run  `dotnet new -t web`, you'll need to upgrade to the current version of .NET Core. Install [.NET Core](https://microsoft.com/net/core)
```

Comment:
check the new rewrite. Latter/former are hard to follow. Also, you can complete this tutorial with the older version using the`-t` as so many have reported. We might want to encourage them to upgrade, but it's not required.